### PR TITLE
Add returnType documentation on paginated operation

### PR DESCRIPTION
**Pull request by jderusse**

### CODE REVIEW

File: docs/clients/ssm.md [hunk 1]
```diff
@@ -23,7 +23,6 @@ $parameters = $ssm->getParametersByPath(new GetParametersByPathRequest([
 ]));
 
 $secrets = [];
-/** @var Parameter $parameter */
```
Comment (Nyholm) [inline docs/clients/ssm.md:26]: We could also remove use statement for Parameter

File: docs/features/pagination.md [hunk 2]
```diff
@@ -54,8 +52,6 @@ $objects = $s3->listObjectsV2(new ListObjectsV2Request([
     'Bucket' => 'my-company-website',
     'Delimiter' => '/'
 ]));
-
-/** @var AwsObject|CommonPrefix $object */
```
Comment (Nyholm) [inline docs/features/pagination.md:58]: We could also remove use statements in this file too

File: docs/clients/ssm.md (hunk 1 above)
Comment (jderusse) [reply docs/clients/ssm.md:26]: good catch!

**Comment by jderusse** [comment]
Thank you @Nyholm for pointing this issue with psalm.
This is fixed by replacing `|` by `&` => https://psalm.dev/r/1dcf3cd32b

**Comment by Nyholm** [comment]
Cool. That will fix some issues for me at work. 

Thanks.

**Pull request merged**